kinds/dashboard/dashboard_kind.cue

```diff
@@ -24,10 +24,11 @@ lineage: seqs: [
 				title?: string
 				// Description of dashboard.
 				description?: string
-
-				// Version of the current dashboard data
-				revision: int64 | *-1 @grafanamaturity(NeedsExpertReview)
-
+				// This property should only be used in dashboards defined by plugins.  It is a quick check
+				// to see if the version has changed since the last time.  Unclear why using the version property 
+				// is insufficient.  
+				revision?: int64 @grafanamaturity(NeedsExpertReview)
```

Comment:
Also wondering why this property is needed given we have a version property for dashboards.

It almost would make more sense for there to be a "isPlugin" bool vs another separate version tracking property 🤔

Reply:
ya -- many other options, but this schema is really here to reflect the reality we have today

Reply:
definitely some smashing together of concerns here